app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -560,6 +560,8 @@ class EditPatientScreen : BaseScreen<
 
   override fun displayBpPassports(bpPassports: List<BPPassportListItem>) {
     bpPassportsContainer.removeAllViews()
+    bpPassportsContainer.visibility = VISIBLE
```

Comment:
I think just this might not be enough to fix the spacing issue. For example, we call `displayBpPassports` even if there are no BP passports. So, that would end up just toggling the visibility. 

So, we need to toggle this view based on whether we have any BP passports are not.